src/applications/vaos/newAppointmentFlow.js

```diff
@@ -76,11 +76,22 @@ export default {
   visitType: {
     url: '/new-appointment/choose-visit-type',
     previous: 'vaFacility',
+    // Update this when reasonForAppointment is merged
     next: 'contactInfo',
   },
+  reasonForAppointment: {
```

Comment:
These two pages don't exist yet, just adding them so I could hook up the Edit links.